src/Console/Command/ScopeHintCommand.php

```diff
@@ -54,15 +63,53 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $scopeValues = $this->scopeHintService->getConfigValuesForScopes($input->getArgument('config_path'));
+
+        $all_configs = $this->scopeHintService->getAllScopes();
```

Comment:
`$all_configs` should be `$allConfigs`. 
(There are more variables in snake case here - all PHP variables should be camel case instead.)

Reply:
This line should be inside the `if` as it's not needed outside.

Reply:
> `$all_configs` should be `$allConfigs`. (There are more variables in snake case here - all PHP variables should be camel case instead.)

Done

Reply:
> This line should be inside the `if` as it's not needed outside.

@avstudnitz But the variable is needed in the if as well in the else block. So I put it outside. If I move it in the if block it also needs to be put into the else block right. Isn't that code duplication that should be avoided?

Reply:
You are right.